client/components/PageLayout/pageLayout.styles.ts

```diff
@@ -0,0 +1,41 @@
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles(theme => ({
+  headerContainer: {
+    'align-items': 'center',
+    backgroundColor: theme.palette.secondary.main,
+    color: 'white',
+    display: 'grid',
```

Comment:
We should use a material-UI grid instead? thoughts?
Ref: https://material-ui.com/components/grid/#grid

Reply:
@vaibhavsingh97 Good call. Done.